tokens-studio/figma-plugin
# Revert manage theme followup

### Why does this PR exist?

Resolve comments in #2883 
Closes #2900




### What does this pull request do?

- Add locales for manage themes
- Remove redundant button styling for manage themes
- Fixes close button being blocked after editing group name
- Adds UI for confirm/cancel buttons while renaming theme group name



### Testing this change

1. Click `Theme: <Theme Name>` then `Manage themes >`
2. Try renaming a group and press `Esc` and confirm that the modal isn't closed
3. Try renaming a group and check that the `Cancel` and `Confirm` buttons work correctly
4. Click through into a theme, click the group name and click `Create new group`
5. Confirm that pressing `Esc` doesn't close the modal
6. Check that all locales are showing/resolving properly
7. In the main modal screen, click to rename a group and confirm that the modal `Close` button works correctly and isn't blocked



### Additional Notes (if any)

**After**

<img width="477" alt="image" src="https://github.com/tokens-studio/figma-plugin/assets/6757532/bc2891a3-a601-40af-b446-ea849a1e2add">
